Source/sources/Castro_geom.cpp

```diff
@@ -1,31 +1,41 @@
-#include "Castro.H"
+#include <Castro.H>
+#include <math.H>
 
 using namespace amrex;
 
 ///
-/// this adds the geometric source term for axisymmetric
-/// coordinates as described in Bernand-Champmartin.  This only
-/// applies to axisymmetric geometry.
+/// this adds the geometric source terms for non-Cartesian Coordinates
+/// This includes 2D Cylindrical (R-Z) coordinate as described in Bernand-Champmartin
+/// and 2D Spherical (R-THETA) coordinate.
 ///
+
 void
-Castro::construct_old_geom_source(MultiFab& source, MultiFab& state_in, Real time, Real dt)
+Castro::construct_old_geom_source([[maybe_unused]]MultiFab& source,
+                                  [[maybe_unused]]MultiFab& state_in,
+                                  [[maybe_unused]]Real time,
+                                  [[maybe_unused]]Real dt)
```

Comment:
I think we decided once that we prefer to use
```
amrex::ignore_unused(state_in);
```

to `[[maybe_unused]]` in the interface